packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -208,6 +208,21 @@ export default function PostOptionsMenu({
     );
   };
 
+  const onUnblockSource = async (): Promise<void> => {
+    const { successful } = await onFollowSource({
+      source: post?.source,
+      requireLogin: true,
+    });
+
+    if (!successful) {
+      return;
+    }
+
+    displayToast(`${post?.source?.name} unblocked`, {
```

Comment:
The label on the button says `Show posts from`, but here we say `unblocked`. Should we change the label or the description here to align?

Reply:
Also follows the same spec as "Don't show" => blocked. We wanted to make a quick win here and will update this if needed as papercut.